compose/ui/ui/src/skikoMain/kotlin/androidx/compose/ui/platform/RenderNodeLayer.skiko.kt

```diff
@@ -203,25 +237,17 @@ internal class RenderNodeLayer(
 
     override fun drawLayer(canvas: Canvas) {
         if (picture == null) {
-            val bounds = size.toSize().toRect()
+            val largestRect = SkRect.makeLTRB(
+                l = PICTURE_MIN_VALUE,
+                t = PICTURE_MIN_VALUE,
+                r = PICTURE_MAX_VALUE,
+                b = PICTURE_MAX_VALUE
+            )
```

Comment:
If you're moving the values to constants, why not move the entire rect?

Reply:
Does it really matter?

Reply:
Less transitions into native code and back, less allocations.

Reply:
It doesn't trigger any native calls

Reply:
Right, I see now. But still, less Java allocations. Not critical.

Reply:
Done